### PR TITLE
New release

### PR DESCRIPTION
**Pull request by Saviour1001**
- Removed Sollet Wallet 
- Updated SDK ref for all Adapters 
- Added `decimals` to chainConfig
- Added `connectedAdapterName`

### CODE REVIEW

File: docs/sdk/web/adapters/coinbase.mdx [hunk 1]
```diff
@@ -101,6 +105,22 @@ If you do pass `chainConfig` in an Adapter, it overwrites the `chainConfig` pass
 
 Checkout the [Coinbase Wallet SDK Documentation](https://docs.cloud.coinbase.com/wallet-sdk/docs/initializing#initializing) for these options.
 
+## Change Adapter Settings
+
+You can change the adapter settings by calling the `setAdapterSettings()` function on the adapter instance.
+
```
Comment (yashovardhan) [inline docs/sdk/web/adapters/coinbase.mdx:111]: Add all the arguments it takes. An example is not expected in SDK reference.

Table and Interface is needed here. Update this in all adapter documentations.

File: src/common/sdk/web/_connectedAdapterName.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+You can configure a lot of [adapters](/sdk/web/adapters) in the Web3Auth instance. If you have installed one of the adapters, you can also use the
+`connectedAdapterName` to get the currently used adapter.
+
+```javascript
+// Initialized Web3Auth instance
+
+// User is logged in using one of the adapters (metamask, torus-evm, etc.)
+
+const adapterName = web3auth.connectedAdapterName;
+// returns the currently used adapter name (metamask, torus-evm, etc.)
```
Comment (yashovardhan) [inline src/common/sdk/web/_connectedAdapterName.mdx:10]: it is not a function, doesn't return the adapter name. Change this to be - contains the name of the adapter currently in use.

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Some minor changes are required. Alongside that, please add in the introduction of the usage pages of modal and core that there's a variable to get the currently connected adapter.

**Comment by yashovardhan** [comment]
also @Saviour1001 please add modalZIndex value in the uiConfig for the modal package. it has been added today as well.

**Comment by Saviour1001** [comment]
Done

**Review by yashovardhan — APPROVED** [review]
LGTM

**Pull request merged**